ddpui/core/dbt_automation/operations/unpivot.py

```
@@ -5,11 +5,11 @@
 
 from logging import basicConfig, getLogger, INFO
 
-from ddpui.dbt_automation.utils.dbtproject import dbtProject
-from ddpui.dbt_automation.utils.columnutils import quote_columnname
-from ddpui.dbt_automation.utils.interfaces.warehouse_interface import WarehouseInterface
-from ddpui.dbt_automation.utils.columnutils import quote_columnname, quote_constvalue
-from ddpui.dbt_automation.utils.tableutils import source_or_ref
+from ddpui.core.dbt_automation.utils.dbtproject import dbtProject
+from ddpui.core.dbt_automation.utils.columnutils import quote_columnname
+from ddpui.utils.warehouse.old_client.warehouse_interface import WarehouseInterface
+from ddpui.core.dbt_automation.utils.columnutils import quote_columnname, quote_constvalue
```

Ruff (0.15.1)

[error] Redefinition of unused `quote_columnname` from line 9: `quote_columnname` redefined here

Remove definition: `quote_columnname`

(F811)